SU-HSDO/suhumsci
# SHS-5958: Bug: Inconsistent "Add Above" feature

# READY FOR REVIEW

## Summary
- Adds "Add between" buttons to paragraphs on Flexible Pages and Private Pages.

## Need Review By (Date)
2024-12-04

## Urgency
medium

## Steps to Test
1. Login to the site
2. Add a new Flexible Page or Private Page
3. Verify that there are "+ Add" buttons for paragraphs

![Screenshot 2024-12-04 at 11 42 51 AM](https://github.com/user-attachments/assets/3949c8cb-5c68-43c8-a5d1-f0d968b05883)

5. Add a paragraph
6. Verify that there is also an "+Add" button that gets added when hovering over a paragraph to add in between
7. Edit an existing Flexible Page or Private Page
8. Verify that the "+Add" buttons show up there as well

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208992447859410
